doc/api/perf_hooks.md

```diff
@@ -56,11 +56,15 @@ to mark specific significant moments in the Performance Timeline.
 ### `performance.measure(name, startMark, endMark)`
 <!-- YAML
 added: v8.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32651
+    description: allow omitted parameters
```

Comment:
This should describe which parameters are omittable, be punctuated (e.g. ``Make `startMark` and `endMark` parameters optional.``)